tasker/executors/parallel_executor.py

```
@@ -135,7 +135,12 @@ def evaluate_parallel_next_condition(next_condition, results, debug_callback=Non
             if log_callback:
                 log_callback(f"'next=always' found, proceeding to next task")
             return True
-            
+
+        if next_condition == 'never':
+            if log_callback:
+                log_callback(f"'next=never' found, not proceeding to next task")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)